test/spec/spectest.cpp

```diff
@@ -562,117 +571,121 @@ void SpecTest::run(std::string_view Proposal, std::string_view UnitName) {
   };
 
   // Command processing. Return true for expected result.
-  auto RunCommand = [&](const rapidjson::Value &Cmd) {
-    // Line number in wast: Cmd["line"].Get<uint32_t>()
-    if (const auto Type = Cmd.FindMember("type"s); Type != Cmd.MemberEnd()) {
-      switch (resolveCommand(Type->value.Get<std::string>())) {
-      case SpecTest::CommandID::Module: {
-        const auto FileName = (TestsuiteRoot / Proposal / UnitName /
-                               Cmd["filename"].Get<std::string>())
-                                  .u8string();
-        const uint64_t LineNumber = Cmd["line"].Get<uint64_t>();
-        if (const auto Name = Cmd.FindMember("name"); Name != Cmd.MemberEnd()) {
-          // Module has name. Register module with module name.
-          LastModName = Name->value.Get<std::string>();
-        } else {
-          // Instantiate the anonymous module.
-          LastModName.clear();
-        }
-        if (onModule(LastModName, FileName)) {
-          EXPECT_TRUE(true);
+  auto RunCommand = [&](const simdjson::dom::object &Cmd) {
+    std::string_view TypeField = Cmd["type"];
```

Comment:
In the old code, it will check `Type != Cmd.MemberEnd()`. However, the new one doesn't.

Reply:
Will add a check for the same, if whether or not the field "type" exists or not.